tests/integration/grpc_test.go

```diff
@@ -186,6 +186,9 @@ func assertAuthority(t *testing.T, expectedAuthority string, clus *integration.C
 	requestsFound := 0
 	for _, m := range clus.Members {
 		for _, r := range m.RecordedRequests() {
+			if r.FullMethod != "/etcdserverpb.KV/Put" {
```

Comment:
Why ?  Please add a comment.

Reply:
Instead of adding commend, added an argument to filter recorded requests by name. This is required as in this test we want to only consider calls done by test client that uses custom endpoints instead of normal ones. 

This is required for this PR as we migrate member list checks to GRPC, those checks are done after cluster setup so they showed up in authority assert.